packages/mrl/src/getTransferData/getRedeemData.ts

```diff
@@ -0,0 +1,49 @@
+import { type ContractConfig, MrlBuilder } from '@moonbeam-network/xcm-builder';
+import { EvmService, type EvmSigner } from '@moonbeam-network/xcm-sdk';
+import type { EvmChain, EvmParachain } from '@moonbeam-network/xcm-types';
+import { WormholeService } from '../services/wormhole';
+
+export interface GetRedeemDataParams {
+  txId: string;
+  chain: EvmChain | EvmParachain;
+}
+
+export async function getRedeemData({ txId, chain }: GetRedeemDataParams) {
+  // TODO this is just for wormhole
```

Comment:
Do we need this comment?

Reply:
It serves as a reminder that all the logic in this function is considering that the tx that is going to be redeemed, is using wormhole as the provider. 
We don't know how other bridges might work, so I don't know how to properly distinguish here which is the provider being used, so I just left a comment